### PR TITLE
Feature/database structure

### PR DESCRIPTION
**Pull request by EugeneTryF**
Created full db structure. It can be changed in future, but i think it's better to create one migration and add new migrations if we need it, than create migration on adding each entity

### CODE REVIEW

File: src/Clinic.DAL/ClinicDbContext.cs [hunk 1]
```diff
@@ -10,16 +10,38 @@ public class ClinicDbContext : DbContext
 
         public ClinicDbContext(DbContextOptions<ClinicDbContext> options, IConfiguration configuration) : base(options)
         {
+            Database.EnsureDeleted();
```
Comment (VadimZharikov) [inline src/Clinic.DAL/ClinicDbContext.cs]: imagine, a db in production is deleted every time you do a request

**Review by VadimZharikov — CHANGES_REQUESTED** [review]

File: src/Clinic.DAL/ClinicDbContext.cs (hunk 1 above)
Comment (EugeneTryF) [reply src/Clinic.DAL/ClinicDbContext.cs]: i understand, but without this compiler gave me exception on previous migration that table Office is already exists.
when i update database with new migration

Comment (EugeneTryF) [reply src/Clinic.DAL/ClinicDbContext.cs]: ok, it's working without this line

File: src/Clinic.DAL/ClinicDbContext.cs [hunk 2]
```diff
@@ -10,16 +10,37 @@ public class ClinicDbContext : DbContext
...
+        public DbSet<PhotoEntity> Photos { get; set; }
+        public DbSet<DocumentEntity> Documents { get; set; }
+        public DbSet<ResultEntity> Results { get; set; }
+        public DbSet<ServiceEntity> Services { get; set; }
+        public DbSet<ServiceCategoryEntity> ServiceCategories { get; set; }
+        public DbSet<SpecializationEntity> Specializations { get; set; }
 
         protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         {
             optionsBuilder.UseQueryTrackingBehavior(QueryTrackingBehavior.NoTracking);
```
Comment (VadimZharikov) [inline src/Clinic.DAL/ClinicDbContext.cs]: what's the reason not to track context entities

File: src/Clinic.DAL/ClinicDbContext.cs [hunk 3]
```diff
@@ -10,16 +10,37 @@ public class ClinicDbContext : DbContext
...
             System.Diagnostics.Debug.WriteLine(_configuration);
```
Comment (VadimZharikov) [inline src/Clinic.DAL/ClinicDbContext.cs]: why is debug code here

**Review by VadimZharikov — CHANGES_REQUESTED** [review]

File: src/Clinic.DAL/ClinicDbContext.cs (hunk 2 above)
Comment (EugeneTryF) [reply src/Clinic.DAL/ClinicDbContext.cs]: have no clue, just took dbContext pattern from one of my previous projects, also can't remember why i have used it

File: src/Clinic.DAL/ClinicDbContext.cs (hunk 3 above)
Comment (EugeneTryF) [reply src/Clinic.DAL/ClinicDbContext.cs]: same reason as with NoTraking

**Review by Genresal — APPROVED** [review]

**Review by VadimZharikov — APPROVED** [review]

**Review by VadimZharikov — APPROVED** [review]

**Pull request merged**